sdk/communication/Azure.Communication.Administration/src/PhoneNumberAdministrationClient.cs

```diff
@@ -752,16 +808,22 @@ public virtual Response CancelSearch(string searchId, CancellationToken cancella
             }
         }
 
-        /// <summary> Purchases the phone number search. </summary>
-        /// <param name="searchId"> The search id to be purchased. </param>
+        /// <summary> Starts purchases the phone number reservation. </summary>
```

Comment:
Let's add a task in the backlog to update the comments. I see some a couple of small mistakes and also some of them could be more descriptive.